src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -384,6 +384,18 @@ public static String formatDateForInstructorCoursesPage(Instant instant, ZoneId
         return formatInstant(instant, zoneId, "d MMM yyyy");
     }
 
+    /**
+     * Formats a date stamp from an {@code instant} for the instructor's feedback sessions page.
+     * Example: 5 May 2017
+     *
+     * @param instant the instant to be formatted
+     * @param zoneId  the time zone to calculate local date
+     * @return the formatted date stamp string
+     */
+    public static String formatDateForInstructorSessionsPage(Instant instant, ZoneId zoneId) {
```

Comment:
Can you add a `TODO` here to rename `formatDateForInstructorCoursesPage` so that we don't have duplicated code?

You can make another PR to do the refactoring (probably change the name from `formatDateForInstructorCoursesPage` to `formatDateForInstructorPages`) so that we can remove this duplicated method.

Reply:
We don't have to do the renaming here as it would touch a lot of files in this PR that are not related to the recycling bin feature.

Reply:
Sure